File: integrations/java/app/revanced/integrations/youtube/patches/HideSeekbarPatch.java
package app.revanced.integrations.youtube.patches;

import app.revanced.integrations.youtube.settings.Settings;

@SuppressWarnings("unused")
public class HideSeekbarPatch {
    public static boolean hideSeekbar() {
        return Settings.HIDE_SEEKBAR.get();
    }
}

File: patches/src/main/kotlin/app/revanced/patches/youtube/seekbar/hideseekbar/HideSeekbarPatch.kt
package app.revanced.patches.youtube.seekbar.hideseekbar

import app.revanced.patcher.patch.bytecodePatch
import app.revanced.patches.youtube.seekbar.seekbarpatch.hideSeekbarPatch

@Deprecated("Patch was moved to app.revanced.patches.youtube.interaction.seekbar")
val hideSeekbarPatch = bytecodePatch {
    dependsOn(
        hideSeekbarPatch
    )
}
